templates/glance/config/00-config.conf

```diff
@@ -76,3 +91,6 @@ endpoint_id = {{ .EndpointID }}
 {{ if (index . "Region") }}
 region_name = {{ .Region }}
 {{ end }}
+
+[image_import_opts]
```

Comment:
@konan-abhi is this correct or do we need a mandatory `glance-image-import.conf`? Keep in mind this config can be overridden by `customConfigService`

Reply:
There is no mandatory need of image-import.conf file. But while documenting import workflow we need to point to correct config files in the documentation.

Reply:
we already documented the way to change this option is via `customConfigService`, I checked [1] and there's no mention of the default `no_op` we added in `glance-image-import.conf`, so it should still be consistent with what we documented.
We can improve that part in a follow up change if you think we need more changes in terms of doc.
Thanks for confirming that section can stay in `00-config.conf`.

[1] https://github.com/openstack-k8s-operators/glance-operator/tree/main/config/samples/import_plugins